src/renderers/shared/reconciler/__tests__/ReactCompositeComponent-test.js

```diff
@@ -146,7 +146,7 @@ describe('ReactCompositeComponent', function() {
     var anchor = instance.getAnchor();
     var actualDOMAnchorNode = ReactDOM.findDOMNode(anchor);
```

Comment:
Yeah, I think I agree, we don't need any of the `actualDOMAnchorNode` stuff.  But I didn't want to confuse the diff by modifying test logic unnecessarily.